instrumentation/spring/spring-jms/spring-jms-2.0/javaagent/src/test/java/SpringListenerTest.java

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
for java tests our convention is to place them in the same package as the instrumentation so here the test code should be in `io.opentelemetry.javaagent.instrumentation.spring.jms.v2_0`